src/main/java/yoda/Storage.java

```diff
@@ -0,0 +1,112 @@
+package yoda;
+
+import yoda.Deadline;
+import yoda.Event;
+import yoda.Quest;
+import yoda.QuestList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private File data;
+    private final String dir;
+    private String filePath;
+
+    public Storage() {
+        this.dir = System.getProperty("user.dir");
+        this.filePath = dir + File.separator + "YodaData" + File.separator + "data.txt";
+        this.data = new File(this.filePath);
+    }
+
+    public ArrayList<Quest> load() {
+        Path path = Paths.get(dir, "YodaData");
+        boolean directoryExists = Files.exists(path);
```

Comment:
The coding standard suggests that booleans be named to sound like booleans, so perhaps something like doesDirectoryExist?